ansible/group_vars/all

```diff
@@ -166,10 +166,12 @@ nginx:
     adminportal: 8443
   ssl:
     path: "{{ openwhisk_home }}/ansible/roles/nginx/files"
-    cert: "openwhisk-cert.pem"
-    key: "openwhisk-key.pem"
+    cert: "openwhisk-server-cert.pem"
+    key: "openwhisk-server-key.pem"
     password_enabled: false
     password_file: "ssl.pass"
+    client_ca_cert: "openwhisk-client-ca-cert.pem"
+    verify_client: "optional"
```

Comment:
@markusthoemmes , i agree with you, make it default value for `off`, if the value is `off`, skip the test case otherwise execute the test case,  but how can i get the `{{ nginx.ssl.verify_cilent }}` value in `WskBasicJavaTests.scala`?